python/requirements.txt

```diff
@@ -14,6 +14,5 @@
 # limitations under the License.
 #
 flask==0.10.1
-pymongo==2.6.3
 apscheduler==2.1.2
-couchdb==0.9
\ No newline at end of file
+couchdb==1.1
```

Comment:
The Lite plan has limitations and if you go over the threshold you get error 429 from Cloudant. The nodejs cloudant library automatically retries. It seems cloudant has a package for Python doing the same auto-retry https://github.com/cloudant/python-cloudant. Should this be used here instead?